target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -125,6 +125,7 @@ case "$FIRMWARE" in
 		caldata_extract "art" 0x5000 0x844
 		ath10k_patch_mac $(macaddr_add $(cat /sys/class/net/eth0/address) -2)
 		;;
+	tplink,eap245-v1|\
```

Comment:
move block according to sorting.

Reply:
Moved up to come after last "tplink,archer-..." block.